tests/test_basic_ops_list.py

```diff
@@ -475,14 +475,25 @@ def test_set_with_invalid_key() -> None:
         (OmegaConf.create([1, 2]), 0, 1),
         (ListConfig(content=None), 0, TypeError),
         (ListConfig(content="???"), 0, MissingMandatoryValue),
+        (OmegaConf.create([1, 2, 3]), slice(None, None, None), [1, 2, 3]),
+        (OmegaConf.create([1, 2, 3]), slice(1, None, None), [2, 3]),
+        (OmegaConf.create([1, 2, 3]), slice(-1, None, None), [3]),
+        (OmegaConf.create([1, 2, 3]), slice(None, 1, None), [1]),
+        (OmegaConf.create([1, 2, 3]), slice(None, -1, None), [1, 2]),
+        (OmegaConf.create([1, 2, 3]), slice(None, None, 1), [1, 2, 3]),
+        (OmegaConf.create([1, 2, 3]), slice(None, None, -1), [3, 2, 1]),
+        (OmegaConf.create([1, 2, 3]), slice(1, None, -2), [2]),
+        (OmegaConf.create([1, 2, 3]), slice(None, 1, -2), [3]),
+        (OmegaConf.create([1, 2, 3]), slice(1, 3, -1), []),
+        (OmegaConf.create([1, 2, 3]), slice(3, 1, -1), [3]),
     ],
 )
 def test_getitem(lst: Any, idx: Any, expected: Any) -> None:
```

Comment:
Please create a new test for it.
It should take a tandard list ([1,2,3]) and test that the behavior of all the slicing variants is consistent between the standard list and ListConfig.

Reply:
Do you mean dividing it into another test with the test cases already used ?

Reply:
I basically want to ensure getitem behavior is consistent for your new implementation with the behavior of the standard list.

find a way to do it while still testing what is already there :)